packages/node-integration-tests/suites/tracing/auto-instrument-postgres/scenario.ts

```diff
@@ -0,0 +1,22 @@
+import '@sentry/tracing';
+
+import * as Sentry from '@sentry/node';
+import pg from 'pg';
+
+Sentry.init({
+  dsn: 'https://public@dsn.ingest.sentry.io/1337',
+  release: '1.0',
+  tracesSampleRate: 1.0,
+});
+
+const transaction = Sentry.startTransaction({
+  op: 'transaction',
+  name: 'Test Transaction',
+});
+
+Sentry.configureScope(scope => {
+  scope.setSpan(transaction);
+});
+
+const client = new pg.Client();
+client.query('test_query', ['a', 'b'], () => transaction.finish());
```

Comment:
Could we look at nested queries? Also simulate actual queries like `SELECT * ...`? TIL that we literally on instrument `.query`.

Reply:
@AbhiPrasad, updated 👍